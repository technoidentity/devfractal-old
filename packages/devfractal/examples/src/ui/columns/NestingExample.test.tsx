import React from 'react'
import renderer, { ReactTestRenderer } from 'react-test-renderer'
import { Nesting } from './NestingExample'

test('Nesting Example of columns', () => {
  const component: ReactTestRenderer = renderer.create(<Nesting />)
  expect(component.toJSON()).toMatchSnapshot()
})
