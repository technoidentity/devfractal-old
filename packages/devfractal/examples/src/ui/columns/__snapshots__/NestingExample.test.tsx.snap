// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nesting Example of columns 1`] = `
<div>
  <section
    className="section"
  >
    <div
      className="container"
    >
      <h1
        className="title is-4"
      >
        Nesting
      </h1>
      <div
        className="columns"
      >
        <div
          className="column"
        >
          <div
            className="notification is-info has-text-centered"
          >
            First column
          </div>
          <div
            className="columns"
          >
            <div
              className="column"
            >
              <div
                className="notification is-info has-text-centered"
              >
                First nested column
              </div>
            </div>
            <div
              className="column"
            >
              <div
                className="notification is-info has-text-centered"
              >
                Second nested column
              </div>
            </div>
          </div>
        </div>
        <div
          className="column"
        >
          <div
            className="notification has-text-centered is-danger"
          >
            Second column
          </div>
          <div
            className="columns"
          >
            <div
              className="column"
            >
              <div
                className="notification is-danger has-text-centered"
              >
                50%
              </div>
            </div>
            <div
              className="column"
            >
              <div
                className="notification is-danger has-text-centered"
              >
                Auto
              </div>
            </div>
            <div
              className="column"
            >
              <div
                className="notification is-danger has-text-centered"
              >
                Auto
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </section>
</div>
`;
